docs/arch/routes.rst

```diff
@@ -38,20 +42,10 @@ Gafaelfawr supports the following routes:
     Retrieves a JWT given an OpenID Connect authorization code obtained via an authentication request.
     The parameters to this route are those for the Token Request in the Authorization Code Flow in `OpenID Connect`_.
 
-``/auth/tokens``
-    Displays all user-issued tokens for the authenticated user.
-
 ``/auth/tokens/influxdb/new``
```

Comment:
The code for that has been temporarily deleted because it was incompatible with the new token scheme, but will be replaced with the new UI that I'm currently working on before the new Gafaelfawr is deployed anywhere.

Reply:
Sorry, maybe it wasn't obvious that the reference was to the InfluxDB tokens.

Reply:
Oh! Yes, this will eventually be incorporated into the token API, but a bit farther down the road. Meanwhile, we're using it in a few specific places, so I left the old method in place for compatibility.